Formula/autorestic.rb

```diff
@@ -0,0 +1,38 @@
+class Autorestic < Formula
+  desc "High level CLI utility for restic"
+  homepage "https://autorestic.vercel.app/"
+  url "https://github.com/cupcakearmy/autorestic/archive/v1.0.8.tar.gz"
+  sha256 "0b27fbc39586471c63f9943029c0c082ba425befb17ff6f7172f10f8941c9210"
+  license "Apache-2.0"
+  head "https://github.com/cupcakearmy/autorestic.git"
+
+
+  depends_on "go" => :build
+  depends_on "restic"
+
+  def install
+    system "go", "build", *std_go_args, "./main.go"
+    
+    (bash_completion/"autorestic").write Utils.safe_popen_read("#{bin}/autorestic", "completion", "bash")
+    (zsh_completion/"_autorestic").write Utils.safe_popen_read("#{bin}/autorestic", "completion", "zsh")
+    (fish_completion/"autorestic.fish").write Utils.safe_popen_read("#{bin}/autorestic", "completion", "fish")
+  end
+
+  test do
+    require 'yaml'
+    # Create sample config
+    config = { 
+      "locations" => { "foo" => { "from" => "repo", "to" => ["bar"] } },
+      "backends" => { "bar" => { "type" => "local", "key" => "secret", "path" => "data" } }
+    }
+    File.open(testpath/".autorestic.yml", "w") { |file| file.write(config.to_yaml) }
+    # Test data
+    (testpath/"repo"/"test.txt").write("This is a testfile")
+    # Backup
+    system "#{bin}/autorestic", "backup", "-a"
+    # Restore
+    system "#{bin}/autorestic", "restore", "-l", "foo", "--to", "restore"
+    # Assert same
+    assert compare_file testpath/"repo"/"test.txt", testpath/"restore"/"test.txt"
```

Comment:
```suggestion
    (testpath/"repo"/"test.txt").write("This is a testfile")
    system "#{bin}/autorestic", "backup", "-a"
    system "#{bin}/autorestic", "restore", "-l", "foo", "--to", "restore"
    assert compare_file testpath/"repo"/"test.txt", testpath/"restore"/"test.txt"
```

The comments aren't necessary here.